### DISCUSSION TITLE
More in-depth tutorials

### DISCUSSION THREAD

**Discussion by victorleblais** [voxeltycoon/issues#529] [General]
_Suggestion inspired by several members of the Discord server: Otter#8522, T3000p#6425 and others_

The first tutorial explains the basics of the game and players could feel more comfortable if there were more tutorials to complete.

Here are some example of topics:
- Basic train setup (like the default one, but with trains)
- Signalling
- Advanced road setup (teaching how to optimise station placement to avoid traffic jams)

If you have any ideas for a tutorial, please share it ! :)

**Comment by madamehardy** [comment]
Managing Routes: discusses how to create route labels, how to assign trucks to labeled routes, and how to edit an existing labeled route so that all trucks on that route follow the new path.

**Comment by asiaclonk** [comment]
First factory setup to build wooden planks: teaches the limitations of conveyor devices and storage buildings (what types of building can accept connectors, storage can only hold one type) as well as the importance of setting accepted items for both storage and arriving vehicles that collect the products (and not the base materials) meant for their destination.
